### DISCUSSION TITLE
How to balance the map?

### DISCUSSION THREAD

**Discussion by ThureHauser** [freeplane/freeplane#1999] [Help]
Hi, I am new to freeplane, but has used a number of other mindmapping softwares, especially MindManager. 
I cannot get freeplane to "balance" the map (as it is called in MindManager), neither automatically nor manually: distributing the topics/children to either side of the core.  I can move individual nodes separately, but that is tedious and doesnt look good. Clicking the "child node layout" doesnt lead me to what i want. And i have searched in the Handbook/documentation without success - probably because "balancing" is called something else in freeplane.  Can anyone help me  ?  And thanks for an otherwise very nice software !!!  Thure

**Comment by euu2021** [comment]
First, check if this is enabled: Preferences…->Behaviour->Distribute new child nodes equally to both sides

Then, see if this solves your problem: https://sourceforge.net/p/freeplane/discussion/758437/thread/b2e5c07381/#2d6c

**Comment by ThureHauser** [comment]
Thanks a lot for the help !
Now i just need to become used to the difference between a child of the
root and a sibling to a child ... a bit confusing, though, at first sight.
But i guess it can be useful when i get used to it :-)

Thure

Den tirs. 27. aug. 2024 kl. 20.50 skrev euu ***@***.***>:

> First, check if this is enabled: Preferences…->Behaviour->Distribute new
> child nodes equally to both sides
>
> Then, see if this solves your problem:
> https://sourceforge.net/p/freeplane/discussion/758437/thread/b2e5c07381/#2d6c
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/freeplane/freeplane/discussions/1999#discussioncomment-10466685>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7Z6VHBIQJZPBJB3XLTSAYDZTTC6HAVCNFSM6AAAAABNFZHSPKVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANBWGY3DQNI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Thure Hauser, 2011 4469
